bin/ns-bundle

```diff
@@ -123,15 +123,20 @@ function gradlewClean() {
 function getTnsVersion() {
     return new Promise((resolve, reject) => {
         const childProcess = spawn("tns", ["--version"], { shell: true });
-
-        childProcess.stdout.on("data", version => resolve(version.toString()));
+        let stdoutData = "";
+        childProcess.stdout.on("data", data => stdoutData += data);
 
         childProcess.on("close", code => {
             if (code) {
                 reject({
                     code,
                     message: `child process exited with code ${code}`,
                 });
+            } else {
+                const versionRegex = /^(?:\d+\.){2}\d+.*?$/m;
+                const matches = stdoutData.toString().match(versionRegex);
```

Comment:
We have a dependency to 'semver'. We can use it instead if the regex.

Reply:
The idea here is to get the version from the string. Output of `tns --version` is:
```
Support for Node.js 9.0.0 is not verified. This CLI might not install or run properly.

3.3.0
```

So the regex should find the `3.3.0` version from it.

Reply:
Makes sense :+1: